modules/runtime/nuxeo-stream/src/main/java/org/nuxeo/lib/stream/log/chronicle/ChronicleRetentionListener.java

```diff
@@ -47,6 +47,10 @@
 
     protected long purgedStamp;
 
+    // NXP-29161 https://stackoverflow.com/questions/58733295/why-does-chronicle-queue-hold-on-to-files-for-a-second-after-close
+    protected final static String OS = System.getProperty("os.name").toLowerCase();
+    public final static boolean IS_WIN = OS.startsWith("win");
```

Comment:
Don't reinvent this, use `import org.apache.commons.lang3.SystemUtils` and check with `SystemUtils.IS_OS_WINDOWS`

Reply:
nuxeo-stream does not currently have a dependency over common-lang3.
=> preserving minimal dependencies for this library.